js/pro/okx.js

```diff
@@ -488,6 +488,7 @@ module.exports = class okx extends okxRest {
                 const update = data[i];
                 const orderbook = this.orderBook ({}, limit);
                 this.orderbooks[symbol] = orderbook;
+                orderbook['symbol'] = symbol;
                 this.handleOrderBookMessage (client, update, orderbook, messageHash);
```

Comment:
Should we add symbol to `handleOrderBookMessage` function (like `parseOrderBook`), eg. `handleOrderBookMessage(client, symbol, message, orderbook, messageHash)`?

Also, was there `symbol` if channel is 'update' (books-l2-tbt)?

Reply:
@sc0Vu not sure I correctly understood the question, but we only need to set the `symbol` once not per update